lua/modules/cubzh.lua

```diff
@@ -12,14 +12,20 @@ Config = {
 	Items = {},
 }
 
--- local WATER_ALPHA = 220
+Dev.DisplayFPS = true
```

Comment:
I think we need to remove this

Reply:
@caillef Oh yes, I think we can merge anyway, I'm clearly not done with work in that file.